<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewKey</class>
 <widget class="QWizard" name="CreateNewKey">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать новый ключ</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="options">
   <set>QWizard::HaveFinishButtonOnEarlyPages|QWizard::HaveNextButtonOnLastPage|QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="check_auto_or_yourself">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_auto_create">
         <property name="text">
          <string>Создать ключ автоматически</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_auto_create">
         <property name="text">
          <string>Создать ключ автоматически</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_yourself">
         <property name="text">
          <string>Ввести ключ с клавиатуры</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_yourself">
         <property name="text">
          <string>Ввести ключ с клавиатуры</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="create_auto_or_yourself">
   <widget class="QLabel" name="area_auto_create">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_area_auto_create">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещайте указатель мыши с зажатой левой клавишей</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="area_create_yourself">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_area_yourself_create">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите свой ключ в поле ниже</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_progress_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgressBar" name="progressBar_create_key">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_progress_bar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Для лучшего шифрования заполните до 100%</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_progress_bar</zorder>
   <zorder>area_auto_create</zorder>
   <zorder>label_area_auto_create</zorder>
   <zorder>area_create_yourself</zorder>
   <zorder>label_area_yourself_create</zorder>
  </widget>
  <widget class="QWizardPage" name="save_and_exit">
   <widget class="QPushButton" name="save_key_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="show_your_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_show_your_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваш ключ шифрования</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
